fdbserver/VersionedBTree.actor.cpp

```diff
@@ -2495,15 +2556,10 @@ class DWALPager final : public IPager2 {
 		return id;
 	}
 
-	Reference<ArenaPage> newPageBuffer(size_t size = 1) override {
-		return Reference<ArenaPage>(new ArenaPage(logicalPageSize * size, physicalPageSize * size));
+	Reference<ArenaPage> newPageBuffer(size_t blocks = 1) override {
+		return ArenaPage::construct(logicalPageSize * blocks, physicalPageSize * blocks);
```

Comment:
T:construct() creates a reference to T? That naming isn't super intuitive to me but I don't really have a better suggestion

Reply:
may be T::allocate() given it instantiates new reference and return to the caller?

Reply:
Yes it does, after discussion earlier today though I'm going to change it to `makeReference` which is stylistically similar to `std::make_..` things.